tools/models/memento/tests/test_cube_builder.py

```diff
@@ -15,13 +15,13 @@ def test_cube_builder_regression() -> None:
     cd tools/models/memento/
 
     python tests/fixtures/census_fixture.py \
-    s3://cellxgene-data-public/cell-census/2023-10-23/soma/census_data/homo_sapiens \
+    s3://cellxgene-data-public/cell-census/2023-10-30/soma/census_data/homo_sapiens \
```

Comment:
Don't use `cellxgene-data-public` for tests and use one of the LTS versions, otherwise these won't be replicable.

Reply:
@ebezzi - These are test fixtures generated from a census version (the date) and stored in the repo. So we can replicate test results. Moreover, my understanding from @atolopko-czi is that we do want to keep in sync with census schema changes. If we need to regenerate the fixture, then we might as well move to more recent census version to generate the fixture and deal with data/schema problems right then and there if the requirement is to keep reasonably up-to-date with census evolution.

Am I misunderstanding something?

Reply:
Yes, that was the idea, @prathapsridharan. But the discussion does make me think that we should have 2 tests:  1) test to validate the diff expr computation, using (possibly older) Census data, and 2) an "acceptance" test that ensures diff expr runs on the latest census, but doesn't care about the correctness of computations, just that it runs to completion.    This regression test addresses (1). Note that "acceptance" tests are run prior to cutting new Census data and api releases, and we expect such tests to hit "real" data.

Reply:
@ebezzi @atolopko-czi - Are y'all ok if we figure out what data version specific test types should depend on when work starts on this ticket: https://github.com/chanzuckerberg/cellxgene-census/issues/977

I think these are valid points but it doesn't need to hold up merging this PR